tokenExchange/src/app/tokenGen.js

```diff
@@ -50,30 +39,37 @@ function getTokenComponent(decodedToken) {
     return token_components;
 }
 
-Date.prototype.addMinutes= function(minutes){
-    var date = new Date(this.getTime());
-    date.setMinutes(date.getMinutes() + minutes);
-    return date;
-}
```

Comment:
Cosa succede alle 16:59 ?

Reply:
Se il TTL è in minuti e non in secondi devo aggiornare il significato delle variabili d'ambiente in documentazione